client/src/Components/Layout/SignUp.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import {
+  Box,
+  Grid,
+  TextField,
+  Button,
+  Link,
+  Typography,
+} from "@material-ui/core";
+import { Person, AlternateEmail, Phone, Lock } from "@material-ui/icons";
+import { withStyles } from "@material-ui/core/styles";
+
+class SignUP extends Component {
```

Comment:
here the component name is SignUp 
but when you export it you write signUp   `export default signUp;`
its should be the SAME